data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/SimpleContingencyAdder.java

```diff
@@ -19,11 +20,12 @@
 /**
  * @author Peter Mitri {@literal <peter.mitri at rte-france.com>}
  */
-public class ComplexContingencyAdder extends AbstractIdentifiableAdder<ContingencyAdder> implements ContingencyAdder {
+public class SimpleContingencyAdder extends AbstractIdentifiableAdder<ContingencyAdder> implements ContingencyAdder {
```

Comment:
you made it more complex and called it SimpleContingencyAdder ? :thinking: 

Maybe ContingencyAdderImpl ?

I was thinking it would be better to have two Adders for the two Contingency implementation. But it would require to have two interfaces (ContingencyAdder and XnodeCongencyAdder) in the crac-api and also one other interface (Contingency + XnodeContingency), which is maybe heavy for something which could be dodged once the refacto of the Crac will be made.

So OK... not a big fan of this ... but it's more a consequence of all the technical debt that we have on the Crac than a consequence of the PR. Once again I don't see a better solution which does not change everything xD.

Reply:
OK I'll rename it